windows/src/global/delphi/general/keymanversion.inc

```diff
@@ -0,0 +1 @@
+  SKeymanVersion = '11.0';
```

Comment:
Is it possible to combine this with `windows/src/version.txt`?
Or will this need to be updated every major release?

Reply:
Ah, that was not supposed to be added -- missed that, sorry. It's gone now.